### ISSUE TITLE
Optimization of Gaussian Process Hyperparameters using Rprop

### ISSUE THREAD

**Issue by sile** [sile/readings#33]
URL
-----

http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.650.6656&rep=rep1&type=pdf

Abstract
----------

> Gaussian processes are a powerful tool for non-parametric regression. Training can be realized by maximizing the likelihood of the data given the model. We show that Rprop, a fast and accurate gradient-based optimization technique originally designed for neural network learning,
can outperform more elaborate unconstrained optimization methods on real world data sets, where it is able to converge more quickly and reliably to the optimal solution.

Table of Contents
--------------------

- (1) Gaussian Process Regression
  - (1-1) Model selection and hyperparameters
  - (1-2) Marginal likelihood
- (2) The Rprop algorithm
- (3) Experiments
  - (3-1) Synthetic data
  - (3-2) Housing data set
  - (3-3) Mauna Loa CO2 concentration
- (4) Results
- (5) Conclusions

BibTeX
------

```
@inproceedings{blum2013optimization,
  title={Optimization of Gaussian process hyperparameters using Rprop.},
  author={Blum, Manuel and Riedmiller, Martin A},
  booktitle={ESANN},
  pages={339--344},
  year={2013},
  organization={Citeseer}
}
```

See Also
---------

- GitHub: https://github.com/mblum/libgp
